### ISSUE TITLE
[Mapping] Clean up the Grammar

### ISSUE THREAD

**Issue by temanbrcom** [eclipse/che-che4z-lsp-for-cobol#341]
## Description ##
As a USER, I would like to have the grammar clean so I can see the syntax error messages clear and descriptive.

## Acceptance Criteria ##
- Unnecessary rules removed
- Technical nodes are hidden

## Definition of Done ##
- The story is implemented according to the description.
- Unit testing done (Preferably by TDD approach, coverage should be at least 70%).
- Code review done by at least one other team member.
- Manual testing is done.
- Documentation is added and reviewed (if needed).
- The pull request is approved.
- The story is merged into the development branch.
- Tests are performed on the development branch.

**Comment by temanbrcom** [comment]
Blocked by #220
